packages/dbml-core/__tests__/parser/mysql-parse/input/delimiter.in.sql

```diff
@@ -0,0 +1,23 @@
+DELIMITER ;;
+
+CREATE TABLE Table_1 (
+  id INT
+);;
+
+DELIMITER @@
+
+CREATE TABLE Table_2 (
+  id INT
+)@@
+
+DELIMITER @
+
+CREATE TABLE Table_3 (
+  id INT
+)@
+
+DELIMITER ;
+
+CREATE TABLE Table_4 (
+  id INT
+);
```

Comment:
Can you add more test for the following cases:
1. `DELIMITER GO`
2. `DELIMITER END`
3. DELIMITER inside comment (should be ignored) `-- DELIMITER ...`
4. DELIMITER appear in quoted string
```
CREATE TABLE Table_5 (
  id INT COMMENT 'just a comment with semi colon ;',
  name varchar default 'default value with semi colon ;'
);

DELIMITER @

CREATE TABLE Table_6 (
  id INT COMMENT 'just a comment with new delimiter @',
  name varchar default 'default value with new delimiter @'
)@
```